src/buffer.c

```diff
@@ -440,7 +440,7 @@ int git_buf_decode_base85(
 
 		acc += de;
 
-		cnt = (output_len < 4) ? output_len : 4;
+		cnt = (output_len < 4) ? (int)output_len : 4;
```

Comment:
Have we really tested that `output_len` fits into an `int`, though? For all I can see, we have

- verified that `output_len` is greater than `base85_len * 4/5`, which doesn't really limit the maximum value of `output_len`
- verified that `output_len + buf->size` does not overflow `size_t`

So `output_len` _could_ in theory still be bigger than `int`, right?